### PR TITLE
Swap links

### PR DESCRIPTION
**Pull request by 0xjocke**
image

### CODE REVIEW

**Review by jmzwar — APPROVED** [review]

**Comment by jmzwar** [comment]
Approved - but do we happen to have the assets as vectors @bachstatter? Would look a bit nicer.

**Comment by bachstatter** [comment]
They are all vectors except curve since that svg is insane

**Comment by jmzwar** [comment]
Ok yeah it's pretty big I imagine.

**Review by noisekit — APPROVED** [review]

File: v2/components/SwapLinks/SwapLinks.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  it('Render correct links for mainnet', () => {
+    cy.viewport(600, 500);
+    cy.mount(
+      <Box paddingY="7" paddingX="4" bg="navy.900" flex="1">
+        <SwapLinksUi networkId={1} outputCurrencyAddress="susdAddressMainnet" />
+      </Box>
+    );
+    cy.get(`[data-testid=oneInchLink]`)
+      .should('have.attr', 'href')
+      .and('eq', 'https://app.1inch.io/#/1/unified/swap/ETH/sUSD');
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/SwapLinks/SwapLinks.cy.tsx:23]: Sorry a bit quick on the merge btn. Hmm the logic i want to test is that correct URL get generated based on the networkId / sUSD address?

[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/SwapLinks/SwapLinks.cy.tsx:23]: But then I don't test the logic? The path/query params is what's dynamic based on the props

**Pull request merged**